### PR TITLE
Fix OSS-Fuzz build status cron after json format update.

### PR DESCRIPTION
**Pull request by inferno-chromium**
Fixes https://github.com/google/oss-fuzz/issues/4278

### CODE REVIEW

**Comment by inferno-chromium** [comment]
> awesome!

Thanks for reviewing!

File: src/appengine/handlers/cron/oss_fuzz_build_status.py [hunk 1]
```diff
@@ -185,19 +184,26 @@ def send_reminder(issue_tracker, issue_id, build_id):
...
+      project_name = project['name']
+      builds = project['history']
+      if not builds:
+        continue
 
+      build_failure = get_build_failure(project_name, build_type)
       if not build_failure:
         continue
 
+      build = builds[0]
```
Comment (Dor1s) [inline src/appengine/handlers/cron/oss_fuzz_build_status.py:203]: while trying to understand why this part is needed, I realized that we might want to run the cron more often now, given that more frequent builds are supported. It runs daily, we can probably run it 4 times a day now?

Comment (inferno-chromium) [reply src/appengine/handlers/cron/oss_fuzz_build_status.py:203]: Sure, feel free to make it 4 times.

**Pull request merged**